catalogs/hadoop-common/src/main/java/org/apache/gravitino/catalog/hadoop/fs/CredentialUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.gravitino.catalog.hadoop.fs;
+
+import java.util.Map;
+
+/** Utility class for Gravitino Virtual File System. */
+public class CredentialUtils {
+
+  /**
+   * Check if Gravitino credential vending is enabled.
+   *
+   * @param config The configuration for Gravitino Virtual File System.
+   * @return true if Gravitino credential vending is enabled, false otherwise.
+   */
+  public static boolean enableGravitinoCredentialVending(Map<String, String> config) {
```

Comment:
The name is little odd,  what you want to check is it happens in Gravitino server side or fileset client side, but from the name, it may happen on both side.

Reply:
I've changed this as suggested.